libs/sdk-backend-mockingbird/api/sdk-backend-mockingbird.api.md

```diff
@@ -71,7 +70,6 @@ export type CompositeBackendPart = {
 // @internal (undocumented)
 export type DashboardRecording = {
     obj: IDashboardWithReferences;
-    alerts: IWidgetAlert[];
```

Comment:
Why total removal of the support for alerting? (I think it's planned for Q2)

Reply:
Tiger is not supporting alerts

Reply:
We have git right ?